cmip6_downscaling/workflows/paths.py

```diff
@@ -1,46 +1,54 @@
-from typing import List, Optional, Tuple, Union
+from dataclasses import dataclass
```

Comment:
```suggestion
from cmip6_downscaling.workflows.utils import BBox
```

(confirm this import is correct)